src/lib_ccx/params_dump.c

```diff
@@ -18,7 +18,9 @@ void params_dump(struct lib_ccx_ctx *ctx)
 			mprint ("stdin");
 			break;
 		case CCX_DS_NETWORK:
-			if (ccx_options.udpaddr == NULL)
+			if (ccx_options.udpsrc != NULL)
+				mprint ("Network, %s@%s:%d", ccx_options.udpsrc, ccx_options.udpaddr, ccx_options.udpport);
```

Comment:
check for ccx_options.udpaddr to be null is still missing.
I would suggest to put (ccx_options.udpsrc != NULL) in else if that would make sure if (ccx_options.udpaddr == NULL) has failed

Reply:
Please see my reply to your previous comment, which was hidden due to white space changes.

Reply:
Please see my reply to your previous comment, which was hidden due to white space changes.